plugins/com.google.cloud.tools.eclipse.appengine.deploy.ui/src/com/google/cloud/tools/eclipse/appengine/deploy/ui/messages.properties

```diff
@@ -24,4 +24,8 @@ use.custom.versioning=Override generated version
 # Deploy console
 action.stop=Stop
 action.remove=Remove
-job.terminated.template=<terminated> {0}
\ No newline at end of file
+job.terminated.template=<terminated> {0}
```

Comment:
I didn't touch the message. @akerekes may have put the angle brackets, and I think they are suitable for console messages.